### DISCUSSION TITLE
Black Screen

### DISCUSSION THREAD

**Discussion by AdamTurock** [MustardChef/WSABuilds#233] [General]
Thank you for working on this.

During some Apps, like NWN: EE, the screen goes black once the game starts.  I know this is a graphics issue.
- I've tried both Hardware/Software graphics.
- I've also tried NVidia Geforce RTX, AMD Radeon, and Intel Iris.
- I tried the latest build: 13.0.7
- I tried various Android graphics debug settings

What would my next steps be?

**Comment by AdamTurock** [comment]
Perhaps this is a Codec issue?

https://www.reddit.com/r/WSA/comments/wsa81s/game_not_rendering_a_video_usm/

**Comment by MustardChef** [comment]
Please open a GitHub issue on this repo. Thank you
